ee/clickhouse/views/test/test_clickhouse_groups.py

```diff
@@ -101,6 +101,7 @@ def test_related_groups(self):
                     "created_at": "2021-05-10T00:00:00Z",
                     "distinct_ids": ["1", "2"],
                     "id": "01795392-cc00-0003-7dc7-67a694604d72",
+                    "uuid": "01795392-cc00-0003-7dc7-67a694604d72",
```

Comment:
Also added this, i.e. it's not exactly the same as the previous behaviour but I don't think it's and issue _famous last words_